freud/density.pyx

```diff
@@ -521,19 +523,20 @@ cdef class GaussianDensity(Compute):
         cdef vec3[uint] width_vector
         if isinstance(width, int):
             width_vector = vec3[uint](width, width, width)
-        elif isinstance(width, (list, tuple)):
+        elif isinstance(width, Sequence):
             if len(width) == 2:
                 width_vector = vec3[uint](width[0], width[1], 1)
             elif len(width) == 3:
                 width_vector = vec3[uint](width[0], width[1], width[2])
             else:
-                raise ValueError("The width must be either a number of pixels "
-                                 "or a list of length 3 (2) indicating the "
-                                 "widths in the 3 (2) spatial dimensions.")
+                raise ValueError("The width must be either a number of bins "
+                                 "or a list indicating the widths in each "
+                                 "spatial dimension (length 2 in 2D, length 3 "
+                                 "in 3D).")
         else:
-            raise ValueError("The width must be either a number of pixels "
-                             "or a list of length 3 (2) indicating the "
-                             "widths in the 3 (2) spatial dimensions.")
+            raise ValueError("The width must be either a number of bins or a "
+                             "list indicating the widths in each " "spatial "
+                             "dimension (length 2 in 2D, length 3 " "in 3D).")
```

Comment:
Fix the weird spacing here between consecutive strings, they're broken up by double quotes for no reason.